Adyen/Core/ToolBar/CancellableToolBar.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+@_spi(AdyenInternal)
+public class CancellableToolBar: ModalToolbar {
```

Comment:
Could be marked `final`

Reply:
Made it final 371694aca888a0dbd865e7232c47d8275294ea30